Expensify/Bedrock
# Return all jobs if mockRequest set as true

We want to return all jobs when adding the mockRequest header so we can automatically mark mocked jobs as completed.

Related to: https://github.com/Expensify/Expensify/issues/75752

# Tests
Test with this PR: https://github.com/Expensify/Bedrock-PHP/pull/86
1. Create job with mockRequest header
2. Run BWM
3. Make sure the job is marked as finished and then deleted but does not run.
